<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en">
<head>
<title>Quantitative Biology  authors/titles &quot;new&quot;</title>
<link rel="shortcut icon" href="/favicon.ico" type="image/x-icon" />
<link rel="stylesheet" type="text/css" media="screen" href="/css/arXiv.css" />
<link rel="alternate" type="application/rss+xml" title="Quantitative Biology " href="http://arxiv.org/rss/q-bio"/>

</head>
<body class="with-cu-identity">
<div id="cu-identity">
<div id="cu-logo">
<a id="insignia-link" href="http://www.cornell.edu/"><img src="/icons/cu/cul_signature_unstyled.gif" alt="Cornell University" width="283" height="76" border="0" /></a>
<div id="unit-signature-links">
<a id="cornell-link" href="http://www.cornell.edu/">Cornell University</a>
<a id="unit-link" href="http://www.library.cornell.edu/">Library</a>
</div>
</div>
<div id="support-ack">
<a href="https://confluence.cornell.edu/x/FYBnDg">We gratefully acknowledge support from<br />the Simons Foundation<br /> and member institutions</a>
</div>
</div>
<div id="header">
<h1><a href="/">arXiv.org</a> &gt; <a href="/list/q-bio/recent">q-bio</a></h1>
<form id="search" method="post" action="/search">
<div class="search-for">Search or Article-id</div>
<div class="links">(<a href="/help">Help</a> | <a href="/find">Advanced search</a>)</div>
<input type="text" name="query" size="24" maxlength="64" />
&nbsp;
<select name="searchtype">
<option value="all" selected="selected">All papers</option>
<option value="ti">Titles</option>
<option value="au">Authors</option>
<option value="abs">Abstracts</option>
<option value="ft">Full text</option>
<option value="help">Help pages</option>
</select>
<input type="submit" value="Go!" /><br />
</form>
</div>
<div id="content">
<div id="dlpage">
<h1>Quantitative Biology </h1>
<h2>New submissions</h2>
<div class="list-dateline">Submissions received from  Thu 28 May 15  to  Fri 29 May 15, announced Mon,  1 Jun 15</div>
<ul>
<li><a href="/list/q-bio/new?skip=0&amp;show=2000">New submissions</a></li>
<li><a href="#item6">Cross-lists</a></li>
<li><a href="#item7">Replacements</a></li>
</ul>
<small>[ total of 10 entries:  <b>1-10</b>  ]</small><br />
<small>[ showing up to 2000 entries per page:  <a href="/list/q-bio/new?skip=0&amp;show=1000">fewer</a> |  <font color="#999999">more</font> ]</small><br />
<h3>New submissions for Mon,  1 Jun 15</h3>
<dl>
<dt><a name="item1">[1]</a>&nbsp;  <span class="list-identifier"><a href="/abs/1505.07855" title="Abstract">arXiv:1505.07855</a> [<a href="/pdf/1505.07855" title="Download PDF">pdf</a>, <a href="/format/1505.07855" title="Other formats">other</a>]</span></dt>
<dd>
<div class="meta">
<div class="list-title">
<span class="descriptor">Title:</span> Genetically Improved BarraCUDA
</div>
<div class="list-authors">
<span class="descriptor">Authors:</span> 
<a href="/find/q-bio/1/au:+Langdon_W/0/1/0/all/0/1">W. B. Langdon</a>, 
<a href="/find/q-bio/1/au:+Lam_B/0/1/0/all/0/1">Brian Yee Hong Lam</a>
</div>
<div class="list-comments">
<span class="descriptor">Comments:</span> UCL, Department of Computer Science, technical report RN/15/03
</div>
<div class="list-subjects">
<span class="descriptor">Subjects:</span> <span class="primary-subject">Genomics (q-bio.GN)</span>

</div>
<p>BarraCUDA is a C program which uses the BWA algorithm in parallel with nVidia
CUDA to align short next generation DNA sequences against a reference genome.
The genetically improved (GI) code is up to three times faster on short paired
end reads from The 1000 Genomes Project and 60percent more accurate on a short
BioPlanet.com GCAT alignment benchmark. GPGPU Barracuda running on a single K80
Tesla GPU can align short paired end nextgen sequences up to ten times faster
than bwa on a 12 core CPU.
</p>
</div>
</dd>
<dt><a name="item2">[2]</a>&nbsp;  <span class="list-identifier"><a href="/abs/1505.07866" title="Abstract">arXiv:1505.07866</a> [<a href="/pdf/1505.07866" title="Download PDF">pdf</a>, <a href="/format/1505.07866" title="Other formats">other</a>]</span></dt>
<dd>
<div class="meta">
<div class="list-title">
<span class="descriptor">Title:</span> Learning universal computations with spikes
</div>
<div class="list-authors">
<span class="descriptor">Authors:</span> 
<a href="/find/q-bio/1/au:+Thalmeier_D/0/1/0/all/0/1">Dominik Thalmeier</a>, 
<a href="/find/q-bio/1/au:+Uhlmann_M/0/1/0/all/0/1">Marvin Uhlmann</a>, 
<a href="/find/q-bio/1/au:+Kappen_H/0/1/0/all/0/1">Hilbert J. Kappen</a>, 
<a href="/find/q-bio/1/au:+Memmesheimer_R/0/1/0/all/0/1">Raoul-Martin Memmesheimer</a>
</div>
<div class="list-comments">
<span class="descriptor">Comments:</span> Main article: 24 pages, 3 figures. Supplemental information: 14 pages, 4 figures
</div>
<div class="list-subjects">
<span class="descriptor">Subjects:</span> <span class="primary-subject">Neurons and Cognition (q-bio.NC)</span>

</div>
<p>Providing the neurobiological basis of information processing in higher
animals, spiking neural networks must be able to learn a variety of complicated
computations, including the generation of appropriate, possibly delayed
reactions to inputs and the self-sustained generation of complex activity
patterns, e.g. for locomotion. Many such computations require previous building
of intrinsic world models. Here we show how spiking neural networks may solve
these different tasks. Firstly, we derive constraints under which classes of
spiking neural networks lend themselves to substrates of powerful general
purpose computing. The networks contain dendritic or synaptic nonlinearities
and have a constrained connectivity. We then combine such networks with
learning rules for outputs or recurrent connections. We show that this allows
to learn even difficult benchmark tasks such as the self-sustained generation
of desired low-dimensional chaotic dynamics or memory-dependent computations.
Furthermore, we show how spiking networks can build models of external world
systems and use the acquired knowledge to control them.
</p>
</div>
</dd>
<dt><a name="item3">[3]</a>&nbsp;  <span class="list-identifier"><a href="/abs/1505.08009" title="Abstract">arXiv:1505.08009</a> [<a href="/pdf/1505.08009" title="Download PDF">pdf</a>, <a href="/format/1505.08009" title="Other formats">other</a>]</span></dt>
<dd>
<div class="meta">
<div class="list-title">
<span class="descriptor">Title:</span> The effect of non-reversibility on inferring rooted phylogenies
</div>
<div class="list-authors">
<span class="descriptor">Authors:</span> 
<a href="/find/q-bio/1/au:+Cherlin_S/0/1/0/all/0/1">S. Cherlin</a>, 
<a href="/find/q-bio/1/au:+Nye_T/0/1/0/all/0/1">T. M. W. Nye</a>, 
<a href="/find/q-bio/1/au:+Boys_R/0/1/0/all/0/1">R. J. Boys</a>, 
<a href="/find/q-bio/1/au:+Heaps_S/0/1/0/all/0/1">S. E. Heaps</a>, 
<a href="/find/q-bio/1/au:+Williams_T/0/1/0/all/0/1">T. A. Williams</a>, 
<a href="/find/q-bio/1/au:+Embley_T/0/1/0/all/0/1">T. M. Embley</a>
</div>
<div class="list-comments">
<span class="descriptor">Comments:</span> 7 figures, 8 supplementary figures
</div>
<div class="list-subjects">
<span class="descriptor">Subjects:</span> <span class="primary-subject">Populations and Evolution (q-bio.PE)</span>

</div>
<p>Most phylogenetic models assume that the evolutionary process is stationary
and reversible. As a result, the root of the tree cannot be inferred as part of
the analysis because the likelihood of the data does not depend on the position
of the root. Yet defining the root of a phylogenetic tree is a key component of
phylogenetic inference because it provides a point of reference for polarising
ancestor/descendant relationships and therefore interpreting the tree. In this
paper we investigate the effect of relaxing the reversibility assumption and
allowing the position of the root to be another unknown quantity in the model.
We propose two hierarchical models which are centred on a reversible model but
perturbed to allow non-reversibility. The models differ in the degree of
structure imposed on the perturbations. The analysis is performed in the
Bayesian framework using Markov chain Monte Carlo methods. We illustrate the
performance of the two non-reversible models in analyses of simulated datasets
using two types of topological priors. We then apply the models to a real
biological dataset, the radiation of polyploid yeasts, for which there is a
robust biological opinion about the root position. Finally we apply the models
to a second biological dataset for which the rooted tree is controversial: the
ribosomal tree of life. We compare the two non-reversible models and conclude
that both are useful in inferring the position of the root from real biological
datasets.
</p>
</div>
</dd>
<dt><a name="item4">[4]</a>&nbsp;  <span class="list-identifier"><a href="/abs/1505.08014" title="Abstract">arXiv:1505.08014</a> [<a href="/pdf/1505.08014" title="Download PDF">pdf</a>]</span></dt>
<dd>
<div class="meta">
<div class="list-title">
<span class="descriptor">Title:</span> Exploitation of the genomic double-strand breaks to reduce the  reproductive power of microorganisms
</div>
<div class="list-authors">
<span class="descriptor">Authors:</span> 
<a href="/find/q-bio/1/au:+Sassi_G/0/1/0/all/0/1">Giandomenico Sassi</a>, 
<a href="/find/q-bio/1/au:+Sassi_N/0/1/0/all/0/1">Nicola Sassi</a>
</div>
<div class="list-comments">
<span class="descriptor">Comments:</span> 13 pages, 1 figure
</div>
<div class="list-subjects">
<span class="descriptor">Subjects:</span> <span class="primary-subject">Genomics (q-bio.GN)</span>; Biological Physics (physics.bio-ph); Biomolecules (q-bio.BM)

</div>
<p>It is shown how to take advantage of the frequent occurrence of double-strand
breaks in the genome of prokaryotic cells, in order to reduce their high
efficient reproductive capability. The analysis examines the physical status of
the free ends of each break and considers how this status can interfere with an
external physical apparatus, with the aim of undermining the repair processes.
We indicate the biological consequences of this interaction and we give an
approximate evaluation of the topological and dynamical effects that arise on
the genomic material involved. The overall result suggests a significant
reduction of the dynamics of the repair.
</p>
</div>
</dd>
<dt><a name="item5">[5]</a>&nbsp;  <span class="list-identifier"><a href="/abs/1505.08125" title="Abstract">arXiv:1505.08125</a> [<a href="/pdf/1505.08125" title="Download PDF">pdf</a>]</span></dt>
<dd>
<div class="meta">
<div class="list-title">
<span class="descriptor">Title:</span> A possible role for stochastic radiation events in the systematic  disparity between molecular and fossil dates
</div>
<div class="list-authors">
<span class="descriptor">Authors:</span> 
<a href="/find/q-bio/1/au:+Melott_A/0/1/0/all/0/1">Adrian L. Melott</a> (University of Kansas)
</div>
<div class="list-comments">
<span class="descriptor">Comments:</span> Accepted for Earth and Life II
</div>
<div class="list-subjects">
<span class="descriptor">Subjects:</span> <span class="primary-subject">Populations and Evolution (q-bio.PE)</span>; Earth and Planetary Astrophysics (astro-ph.EP); High Energy Astrophysical Phenomena (astro-ph.HE); Medical Physics (physics.med-ph)

</div>
<p>Major discrepancies have been noted for some time between fossil ages and
molecular divergence dates for a variety of taxa. Recently, systematic trends
within avian clades have been uncovered. The trends show that the disparity is
much larger for mitochondrial DNA than for nuclear DNA; also that it is larger
for crown fossil dates than stem fossil dates. It was argued that this pattern
is largely inconsistent with incompleteness of the fossil record as the
principal driver of the disparity. A case is presented that given the expected
mutations from a fluctuating background of astrophysical radiation from such
sources as supernovae, the rate of molecular clocks is variable and should
increase into the past. This is a possible explanation for the disparity. One
test of this hypothesis is to look for an acceleration of molecular clocks 2 to
2.5 Ma due to a probable moderately nearby supernova at that time. Another is
to look for reduced disparity in benthic organisms of the deep ocean.
</p>
</div>
</dd>
</dl>
<h3>Cross-lists for Mon,  1 Jun 15</h3>
<dl>
<dt><a name="item6">[6]</a>&nbsp;  <span class="list-identifier"><a href="/abs/1505.08171" title="Abstract">arXiv:1505.08171</a> (cross-list from stat.AP) [<a href="/pdf/1505.08171" title="Download PDF">pdf</a>, <a href="/format/1505.08171" title="Other formats">other</a>]</span></dt>
<dd>
<div class="meta">
<div class="list-title">
<span class="descriptor">Title:</span> An integrative statistical model for inferring strain admixture within  clinical Plasmodium falciparum isolates
</div>
<div class="list-authors">
<span class="descriptor">Authors:</span> 
<a href="/find/stat/1/au:+OBrien_J/0/1/0/all/0/1">John D. O&#x27;Brien</a>, 
<a href="/find/stat/1/au:+Iqbal_Z/0/1/0/all/0/1">Zamin Iqbal</a>, 
<a href="/find/stat/1/au:+Amenga_Etego_L/0/1/0/all/0/1">Lucas Amenga-Etego</a>
</div>
<div class="list-subjects">
<span class="descriptor">Subjects:</span> <span class="primary-subject">Applications (stat.AP)</span>; Populations and Evolution (q-bio.PE)

</div>
<p>Since the arrival of genetic typing methods in the late 1960's, researchers
have puzzled at the clinical consequence of observed strain mixtures within
clinical isolates of Plasmodium falciparum. We present a new statistical model
that infers the number of strains present and the amount of admixture with the
local population (panmixia) using whole-genome sequence data. The model
provides a rigorous statistical approach to inferring these quantities as well
as the proportions of the strains within each sample. Applied to 168 samples of
whole-genome sequence data from northern Ghana, the model provides
significantly improvement fit over models implementing simpler approaches to
mixture for a large majority (129/168) of samples. We discuss the possible uses
of this model as a window into within-host selection for clinical and
epidemiological studies and outline possible means for experimental validation.
</p>
</div>
</dd>
</dl>
<h3>Replacements for Mon,  1 Jun 15</h3>
<dl>
<dt><a name="item7">[7]</a>&nbsp;  <span class="list-identifier"><a href="/abs/1505.01376" title="Abstract">arXiv:1505.01376</a> (replaced) [<a href="/pdf/1505.01376" title="Download PDF">pdf</a>]</span></dt>
<dd>
<div class="meta">
<div class="list-title">
<span class="descriptor">Title:</span> Tissue fibrosis: a principal evidence for the central role of Misrepairs  in aging
</div>
<div class="list-authors">
<span class="descriptor">Authors:</span> 
<a href="/find/q-bio/1/au:+Wang_Michelitsch_J/0/1/0/all/0/1">Jicun Wang-Michelitsch</a>, 
<a href="/find/q-bio/1/au:+Michelitsch_T/0/1/0/all/0/1">Thomas M. Michelitsch</a>
</div>
<div class="list-comments">
<span class="descriptor">Comments:</span> 13 pages, 4 figures
</div>
<div class="list-subjects">
<span class="descriptor">Subjects:</span> <span class="primary-subject">Tissues and Organs (q-bio.TO)</span>

</div>
</div>
</dd>
<dt><a name="item8">[8]</a>&nbsp;  <span class="list-identifier"><a href="/abs/1505.04600" title="Abstract">arXiv:1505.04600</a> (replaced) [<a href="/pdf/1505.04600" title="Download PDF">pdf</a>]</span></dt>
<dd>
<div class="meta">
<div class="list-title">
<span class="descriptor">Title:</span> Spike Pattern Structure Influences Efficacy Variability under STDP and  Synaptic Homeostasis
</div>
<div class="list-authors">
<span class="descriptor">Authors:</span> 
<a href="/find/q-bio/1/au:+Bi_Z/0/1/0/all/0/1">Zedong Bi</a>, 
<a href="/find/q-bio/1/au:+Zhou_C/0/1/0/all/0/1">Changsong Zhou</a>, 
<a href="/find/q-bio/1/au:+Zhou_H/0/1/0/all/0/1">Hai-Jun Zhou</a>
</div>
<div class="list-subjects">
<span class="descriptor">Subjects:</span> <span class="primary-subject">Neurons and Cognition (q-bio.NC)</span>

</div>
</div>
</dd>
<dt><a name="item9">[9]</a>&nbsp;  <span class="list-identifier"><a href="/abs/1505.06726" title="Abstract">arXiv:1505.06726</a> (replaced) [<a href="/pdf/1505.06726" title="Download PDF">pdf</a>, <a href="/format/1505.06726" title="Other formats">other</a>]</span></dt>
<dd>
<div class="meta">
<div class="list-title">
<span class="descriptor">Title:</span> Seasonal states in the evolutionary dynamics of finite populations
</div>
<div class="list-authors">
<span class="descriptor">Authors:</span> 
<a href="/find/q-bio/1/au:+Tkachenko_O/0/1/0/all/0/1">Olena Tkachenko</a>, 
<a href="/find/q-bio/1/au:+Thingna_J/0/1/0/all/0/1">Juzar Thingna</a>, 
<a href="/find/q-bio/1/au:+Denisov_S/0/1/0/all/0/1">Sergey Denisov</a>, 
<a href="/find/q-bio/1/au:+Zaburdaev_V/0/1/0/all/0/1">Vasily Zaburdaev</a>, 
<a href="/find/q-bio/1/au:+Hanggi_P/0/1/0/all/0/1">Peter H&#xe4;nggi</a>
</div>
<div class="list-comments">
<span class="descriptor">Comments:</span> 5 pages + 4 figures + supplementary material included. Comments are welcome
</div>
<div class="list-subjects">
<span class="descriptor">Subjects:</span> <span class="primary-subject">Populations and Evolution (q-bio.PE)</span>; Statistical Mechanics (cond-mat.stat-mech); Biological Physics (physics.bio-ph)

</div>
</div>
</dd>
<dt><a name="item10">[10]</a>&nbsp;  <span class="list-identifier"><a href="/abs/1505.07821" title="Abstract">arXiv:1505.07821</a> (replaced) [<a href="/pdf/1505.07821" title="Download PDF">pdf</a>, <a href="/format/1505.07821" title="Other formats">other</a>]</span></dt>
<dd>
<div class="meta">
<div class="list-title">
<span class="descriptor">Title:</span> Genetic correlations greatly increase mutational robustness and can both  reduce and enhance evolvability
</div>
<div class="list-authors">
<span class="descriptor">Authors:</span> 
<a href="/find/q-bio/1/au:+Greenbury_S/0/1/0/all/0/1">Sam F. Greenbury</a>, 
<a href="/find/q-bio/1/au:+Schaper_S/0/1/0/all/0/1">Steffen Schaper</a>, 
<a href="/find/q-bio/1/au:+Ahnert_S/0/1/0/all/0/1">Sebastian E. Ahnert</a>, 
<a href="/find/q-bio/1/au:+Louis_A/0/1/0/all/0/1">Ard A. Louis</a>
</div>
<div class="list-subjects">
<span class="descriptor">Subjects:</span> <span class="primary-subject">Populations and Evolution (q-bio.PE)</span>

</div>
</div>
</dd>
</dl>
<ul>
<li><a href="/list/q-bio/new?skip=0&amp;show=2000">New submissions</a></li>
<li><a href="#item6">Cross-lists</a></li>
<li><a href="#item7">Replacements</a></li>
</ul>
<small>[ total of 10 entries:  <b>1-10</b>  ]</small><br />
<small>[ showing up to 2000 entries per page:  <a href="/list/q-bio/new?skip=0&amp;show=1000">fewer</a> |  <font color="#999999">more</font> ]</small><br />
</div>
</div>
<div id="footer">

<hr />
<p>Links to:
<a href="/" accesskey="a">arXiv</a>, 
<a href="/form/q-bio">form interface</a>,
<a href="/find/q-bio">find</a>,
<a href="/archive/q-bio">q-bio</a>, <a href="/list/q-bio/recent">recent</a>, <a href="/list/q-bio/1505">1505</a>,
<a href="/help/contact">contact</a>,
<a href="/help/" accesskey="h"><span class="accesskey">h</span>elp</a>&nbsp;
<small>(<a href="/help/accesskeys">Access key</a> information)</small>
</p>
<hr />
</div>
</body>
</html>
